CONTRIBUTING.md

```diff
@@ -9,9 +9,12 @@
 * [Idle Pull Requests Policy](#idle)
 * [Unresolved Pull Requests Policy](#unresolved)
 
+This document is an extension to the generic [Anuket Project Operations and Procedures](https://wiki.anuket.io/display/HOME/Anuket+Project+Operations+and+Procedures) and describes contribution rules to the Anuket specifications, that are the set of documents maintained in this repository. 
+
 <a name="permissions"></a>
 ### Permission levels
-There are different permissions (levels) available for contributing into CNTT:
+
+There are different permissions (levels) available for contributing into the Anuket specifications:
 - **Outside Collaborators**: This includes general public. 
   - Outside collaborators can create issues and add comments to issues and Pull Requests.
 - **Read, Triage, Write, and Maintain** Permissions, which are explained in [here](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization), will be assigned as follows:
```

Comment:
```suggestion
- **Read** will be assigned to those who want to create issues, join the discussion, receive notifications, and provide feedback.

Reply:
This was put in the wrong place.  It refers to the line below that needs to change.

Reply:
@bfcohen100 something is missing in your line above - "read" doesn't allow rights to create

Reply:
I think it allows the creation of issues.

Reply:
@bfcohen100 I do not see this misplaced. Line 20 lists all the non outside collaborators and lines 21-24 explain each of the,